### ISSUE TITLE
KEBA Binding 2.2.0 does not work

### ISSUE THREAD

**Issue by ThomasBrodkorb** [openhab/openhab2-addons#3070]
After an unintended upgrade to 2.2.0 (Openhabian just does an update when doing an apt-get) the KEBA Binding does not work anymore.
After two terrible days, I found a 2.0.0 JAR file for the binding, installed that one manually and now it works again.

**Comment by martinvw** [comment]
Did you see / follow the hints in the release notes?

https://github.com/openhab/openhab-distro/wiki/openHAB-2.2-Release-Notes#keba-binding

**Comment by ThomasBrodkorb** [comment]
No, I did not see that release note. I did not even know that there is useful information - as I said, I did not want to update Openhab at all, so I did not even look thru the release notes. And until now, I did not know where they are.
Anyway, that comment is not of any help.
You can not remove KEBA things (not with paper UI = status is "removing", even after reboot of pi3 - and not with the thing file), not properly uninstall the binding and not properly install the binding ("uninstalling" runs forever, no useful output at the karaf console.)
So I ended up copying the old jar file and all works as expected.
All I can say: Do not modify things that are working, especially not the syntax. If required, allow the new thing type on top of the old one and change the documentation to the new syntax. (https://docs.openhab.org/addons/bindings/keba/readme.html#binding-configuration)

**Comment by martinvw** [comment]
I was wrong, it was in the release notes but the change did not make it in yet, so it's another problem. Any logging is most likely very welcome.

CC: @kgoderis 

FTR: the change is in https://github.com/openhab/openhab2-addons/pull/2844

Feel free to mix in the discussion if you want to debat the change.

**Comment by kgoderis** [comment]
@martinvw @ThomasBrodkorb I just had a look at the code and indeed, with the introduction of https://github.com/openhab/openhab2-addons/commit/c65fa0889bac903a331c79fae58e71de84b34320#diff-b06428a01c765a9d24e5c6faba386802 in all my haste I made a mistake by not changing the moniker for the ThingType in the KebaBindingConstants.java file. I did not notice that problem until today since I run a custom jar (based on #2844) in my environment. In its current state the binding will never work. Mea culpa

@martinvw I suggest we merge #2844 as soon as possible
